homeassistant/components/remootio/exceptions.py

```diff
@@ -0,0 +1,37 @@
+"""Exceptions for the Remootio integration."""
+from aioremootio import RemootioClient, RemootioError
+
+from homeassistant.exceptions import HomeAssistantError
+
+
+class UnsupportedRemootioDeviceError(RemootioError):
+    """Error to indicate unsupported Remootio device."""
+
+    def __init__(
+        self,
+        device: RemootioClient,
+        message="Your device isn't supported, possibly because it hasn't a sensor installed.",
+    ) -> None:
+        """Initialize the instance of this class."""
+
+        super().__init__(device, message)
+
+
+class UnsupportedRemootioApiVersionError(UnsupportedRemootioDeviceError):
```

Comment:
This looks a bit strange. Why are you subclassing the libs exceptions here. These are HomeAssistant exceptions right?

Reply:
Fixed.